apps/docs/components/package.json

```
@@ -8,8 +8,8 @@
   },
   "scripts": {
     "dev:docs": "yarn update-files && concurrently \"vuepress dev\" \"yarn watch-base\" \"yarn watch-blocks\" \"yarn watch-hooks\" \"yarn watch-customization\"",
-    "build": "yarn update-files && vuepress build",
-    "build:docs": "yarn build",
+    "build:not-run": "yarn update-files && vuepress build",
+    "build:docs": "yarn build:not-run",
```

That's soooooo weird, we need to replace turbo with something else quick

---

Btw cannot we just do?
```suggestion
    "build:docs": "yarn update-files && vuepress build",
```

---

https://github.com/vuestorefront/storefront-ui/pull/2637/commits/e5dcddf625ca0318babd1c00a6a7ad83ad32d346